src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java

```diff
@@ -46,32 +45,26 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         CourseRoster roster = new CourseRoster(logic.getStudentsForCourse(courseId),
                                                logic.getInstructorsForCourse(courseId));
         
-        data = new InstructorFeedbackResponseCommentsLoadPageData(account);
-        data.feedbackResultBundles = getFeedbackResultBundles(courseId, fsname, roster);
-        data.instructorEmail = instructor.email;
-        data.currentInstructor = instructor;
-        data.roster = roster;
-        data.feedbackSessionIndex = fsindex;
-        data.numberOfPendingComments = logic.getCommentsForSendingState(courseId, CommentSendingState.PENDING).size() 
+        int numberOfPendingComments = logic.getCommentsForSendingState(courseId, CommentSendingState.PENDING).size() 
                 + logic.getFeedbackResponseCommentsForSendingState(courseId, CommentSendingState.PENDING).size();
+        FeedbackSessionResultsBundle bundle = getFeedbackResultBundle(courseId, fsname, roster);
```

Comment:
can enlighten me why we are able to use `getFeedbackResultBundle` (get a single object instead of map of objects)?